packages/react/src/components/link/Link.tsx

```diff
@@ -106,7 +112,7 @@ export const Link = React.forwardRef<HTMLAnchorElement, LinkProps>(
         href={href}
         style={style}
         {...(openInNewTab && { target: '_blank', rel: 'noopener' })}
-        {...((openInNewTab || external) && { 'aria-label': composeAriaLabel() })}
+        {...((openInNewTab || external || ariaLabel) && { 'aria-label': composeAriaLabel() })}
```

Comment:
It could be like that, although in that case if none of these props are given the aria-label would be the same as link text which sounds redundant and isn't useful for screen reader users.

Reply:
Ok 👍